en/flight_modes/offboard.md

```diff
@@ -0,0 +1,42 @@
+# Offboard Flight Mode
+
+The vehicle obeys a position, velocity or attitude setpoint provided over MAVLink (often from a companion computer connected via serial cable or wifi). The setpoint can be provided by APIs like [DroneCore](http://dronecore.io/) or [MAVROS](https://github.com/mavlink/mavros).
+
+> **Note** 
+>  * This mode requires position information (GPS or optical flow).
+>  * This mode is automatic (RC control is disabled [by default](../advanced_config/parameter_reference.md#COM_RC_OVERRIDE) except to change modes).
+>  * The vehicle must be armed before this mode can be engaged.
+
+## Description
+
+Offboard mode is primarily used for controlling vehicle movement and attitude, and supports only a very limited set of MAVLink commands (more may be supported in future). It can be used to:
+* Control vehicle position or velocity (`SET_POSITION_TARGET_LOCAL_NED`). 
+* Control vehicle attitude/orientation (`SET_ATTITUDE_TARGET`)
+
+> **Note** Velocity and attitude commands must be resent regularly at a rate of around (TBD Hz) or PX4 will stop the vehicle. 
+
+Other operations, like taking off, land, return to launch, arming, are best handled using the appropriate modes. Operations like uploading, downloading missions can be performed in any mode.
+
+Offboard mode requires an active connection to a remote system. If the connection is lost the vehicle will attempt to land or perform some other failsafe action (after a timeout ([COM_OF_LOSS_T](#COM_OF_LOSS_T)). The action is defined in the parameters [COM_OBL_ACT](#COM_OBL_ACT) and [COM_OBL_RC_ACT](#COM_OBL_RC_ACT).

```

Comment:
As I said above, that same connection is established when sending that same setpoint stream.

Reply:
OK, see request for clarification above. What if you're in offboard most but you don't want to go anywhere - how do you keep the offboard mode alive? - Just send setpoints to move to the current position?

Reply:
if you don't want to go anywhere you need to send a setpoint stream with the current position of the vehicle.